dev/NavigationView/NavigationView_InteractionTests/NavigationViewTests.cs

```diff
@@ -383,6 +383,12 @@ public void PaneOpenForceCloseTest()
                 {
                     CheckBox isPaneOpenCheckBox = new CheckBox(FindElement.ById("IsPaneOpenCheckBox"));
 
+                    if (isPaneOpenCheckBox.ToggleState == ToggleState.Off)
```

Comment:
Do you know why you need to make change here? I don't expect it happen here

Reply:
That part is not needed anymore. I will remove it.